src/debug.ts

```diff
@@ -147,20 +158,25 @@ export class DashmipsDebugSession extends LoggingDebugSession {
             } as DashmipsBreakpointInfo
         })
 
-        await this.client.ready()
+        // We need to block here until the socket is open
+        if (!this.client.open) {
+            await this.configurationDone.wait(1000)
+            await this.configurationDone.wait(1100)
+        }
```

Comment:
Same as above, two calls?

Reply:
After testing, I found that the python code needed a bit longer to run if there were no breakpoints set.  However, adding more time to a single wait call for some reason does not solve the problem (I've tried 2000, 3000, etc...) but adding two separate calls works.  Not sure why this happens, if you know why I'd be happy to find an alternative solution.

Reply:
Also I managed to resolve the conflict, but after adding the code again there is another conflict with the master.  Does this mean that the code cannot be ever added/merged due to conflicts?